### PR TITLE
(Feat)Auth Token Exchange Documentation

### PR DESCRIPTION
**Pull request by chanjant**
**NOTE: do not merge until API is complete**

document refresh token endpoint

Ticket Link: https://www.notion.so/immersve/Refresh-token-endpoint-docs-59d3ab39e1ac4a16a2ae243323d3062d?pvs=4

Test deploy url: https://214-merge--jovial-scone-ec740d.netlify.app/api-reference/auth-token-exchange/

### CODE REVIEW

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+post:
+  tags:
+    - authentication
+  summary: Auth Token Exchange
+  description: |
+    The token endpoint returns a new access token for a user without requiring the user to re-authenticate.
```
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:6]:
```suggestion
This endpoint cant be used to refresh an Access Token using the Refresh Token
```

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            refreshToken:
+              type: string
+              description: The token previously issued to the client when login.
```
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:15]:
```suggestion
              description: JWT token that is used to obtain more access tokens.
```

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+              example: "cda232434dsfds342923frENnKthng3ji4.dseyJ1c2VyX2lkIjoidXNlcl8xMjM0NTYiLCJ0b2tlbl9pZCI6InRva2VuXzc4OTEwIiwiaWF0IjoxNjE1MjY5MTIyLCJleHAiOjE2MTU4NzM5MjJ9.Qy4f3hK6S9DqibtV78iqMzT0NSXx8OH8R3qL_UfzpM4km1ujGbnkIjUn"
+            authCode:
```
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:17]: i dont think you need authCode in this version of the docs yet. that is something @immersve/spoink will address in upcoming work

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml (hunk 1 above)
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:6]: we should mention somewhere that theres a 60 grace period on the token when its invalidated

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          properties:
+            refreshToken:
+              type: string
+              description: JWT token that is used to obtain more access tokens.
+              example: "cda232434dsfds342923frENnKthng3ji4.dseyJ1c2VyX2lkIjoidXNlcl8xMjM0NTYiLCJ0b2tlbl9pZCI6InRva2VuXzc4OTEwIiwiaWF0IjoxNjE1MjY5MTIyLCJleHAiOjE2MTU4NzM5MjJ9.Qy4f3hK6S9DqibtV78iqMzT0NSXx8OH8R3qL_UfzpM4km1ujGbnkIjUn"
+            clientApplicationId:
+              type: string
+              description: Id of the client application.
+              example: '29097f50d221858223a17633e36716f9'
+            clientSecret:
```
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:21]: should we remove this if its still pending implementation @javcode

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+post:
+  tags:
+    - authentication
+  summary: Auth Token Exchange
+  description: |
+    This endpoint can be used to request a new access token using a refresh token.
```
Comment (tonykha123) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:6]: should we also mention that this api is recallable for 60 seconds?

Comment (tonykha123) [reply imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:6]: @javcode

File: imsv-docs-docusaurus/openapi/endpoints/login/models/login-response.yaml [hunk 6]
```diff
@@ -6,7 +6,11 @@ properties:
     description: The authentication token
     type: string
     example: "eyJhbGciOiJSUzI1NiIsI..."
+  refreshToken:
+    description: JWT that is used to request a new access token. (Valid for 30 days.)
```
Comment (NicBathgate) [inline imsv-docs-docusaurus/openapi/endpoints/login/models/login-response.yaml:10]:
```suggestion
    description: JWT that is used to request a new access token. Refresh tokens are valid for 30 days.
```

**Review by NicBathgate — APPROVED** [review]
Looking gooooooood!

**Pull request merged**